_events/2016-01-31-feminism-is-for-everybody.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: feminism-is-for-everybody
+slug: feminism-is-for-everybody
+title: "January Book Club"
+date: "2017-01-31 14:00:00"
```

Comment:
It's referred to as `date` in the [Jekyll documentation](https://jekyllrb.com/docs/frontmatter/).